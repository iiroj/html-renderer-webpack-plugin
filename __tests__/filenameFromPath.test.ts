import { filenameFromPath } from "../index";

const paths = ["/", "/foo", "/bar/"];

describe("filenameFromPath", () => {
  it("should generate filenames from paths", () => {
    const filenames = paths.map(path => filenameFromPath(path));
    expect(filenames).toMatchSnapshot();
  });
});
