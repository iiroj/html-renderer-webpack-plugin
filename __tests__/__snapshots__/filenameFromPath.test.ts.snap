// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`filenameFromPath should generate filenames from paths 1`] = `
Array [
  "index.html",
  "foo.html",
  "bar/index.html",
]
`;
